lib/resources/example_json/r4_examples_location.rb

```diff
@@ -0,0 +1,123 @@
+# frozen_string_literal: true
+
+module Cerner
+  module Resources
+    R4_LOCATION_BUNDLE ||= {
```

Comment:
Bundled responses have a few additional fields in comparison to a Read data such as these fields - [NutirtionOrder example](https://github.com/cerner/fhir.cerner.com/blob/main/lib/resources/example_json/r4_examples_nutrition_order.rb#L240-L249)

Can we update to have those fields? we could also do something like here: [r4_examples_nutrition_order](https://github.com/cerner/fhir.cerner.com/blob/main/lib/resources/example_json/r4_examples_nutrition_order.rb#L239)

Reply:
Fix for this:
https://github.com/cerner/fhir.cerner.com/commit/533e50911cc35b3e9313e0184120b17340db3e04